zama-poc/src/main.rs

```diff
@@ -15,7 +20,7 @@ fn main() {
 
     info!("Hello, Zama!");
 
-    let filename = "/Users/vishakh/dev/MonadicDNA/zama-poc/GFGFilteredUnphasedGenotypes23andMe.txt";
+    let filename = "/home/amardeep/MonadicDNA/zama-poc/GFGFilteredUnphasedGenotypes23andMe.txt";
```

Comment:
Could you please make this a relative path that works on all our computers?

Reply:
The text file should be under the zama-poc folder,